src/qilib/configuration_helper/adapters/uhfli_instrument_adapter.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, address: str) -> None:
         self._instrument: ZIUHFLI = ZIUHFLI(self.name, device_ID=address)
 
     def _filter_parameters(self, parameters: PythonJsonStructure) -> PythonJsonStructure:
+        parameters.pop('scope_samplingrate_float', None)
+        parameters.pop('scope_duration', None)
```

Comment:
I really don't like popping these parameters. Maybe they are not used today, but might very well be used in the future and to keep them out of the config is misleading.

There are some helper methods in the qcodes driver, like `setlength`, `setduration` and `setsamplingrate` that might be of some value here.

Reply:
Agree. However one could argue that it's a little performance decrease to save these redundant values. But in this case I don't think it's a strong argument and I prefer simplicity over this 'performance' issue.

After some playing around with the settings, debugging these methods and discussion with @CumulonimbusCalvus we came to the conclusion that there isn't really a problem in parameter (order) sensitivity. There seems to be idempotency when reading and applying config (even though the config might be applied again in reverse or random order). The instrument driver itself already takes care when setting these parameters that have any dependency with other parameters. We don't see any problems as long as parameters are changed via the instrument driver and not by altering the config directly.